modules/data-entry/src/main/java/io/uhndata/cards/DeleteServlet.java

```diff
@@ -365,7 +367,11 @@ private String listReferrersFromTraversal()
                         questionnaires.add(n.getProperty("title").getString());
                         break;
                     default:
-                        otherCount++;
+                        if ("cards/Answer".equals(n.getProperty("sling:resourceSuperType").getString())) {
+                            answerCount++;
```

Comment:
Can't this be moved to a `case`?

Reply:
Let's also add a `case` for `cards:AnswerSection`.

Reply:
It cannot be moved to a case, since the switch is on `primaryNodeType`, which varies from answer to answer, but they have the same supertype. However, I would change this to a safer node type check, since not all nodes may have a `resourceSuperType property`:

```suggestion
                        if (n.isNodeType("cards:Answer")) {
                            answerCount++;
```

Reply:
Right, I see, thanks